graph/Graph/Graph.cpp

```diff
@@ -0,0 +1,93 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#include <string>
+#include <iostream>
+#include "Graph.h"
+
+Graph::Graph(int& size_t, std::vector<std::vector<int>>& graph) : size(size_t) {
+    matrix = new int*[size];
+    visited = new bool[size];
```

Comment:
Rather do not use raw arrays. Preferred are standard containers like:
map, vector, std::array (this is something different then raw array), list, end so on...